src/elements/core/styles/mixins/table.scss

```
@@ -48,6 +52,13 @@
   }
 }
 
+@mixin table-header-row {
+  // If there is more than one header row, only the last one has the bottom border
+  &:not(:last-of-type) > th {
+    border-bottom: none;
```

possible to use logical?
```suggestion
    border-block-end: none;
```

---

It doesn't really add any value since the borders are independent of the writing mode. 
I feel like it only adds the burden to 'test' the table composition in more cases. What do you think?

---

It's more our base rule: always use logical properties and only if not working use not logical ones and comment it

---

I'm still not thoroughly convinced but I don't feel like it's worth enough to discuss it 😅 
I'll change other occurrences too